misc/services/camel/de-metas-camel-externalsystems/core/src/main/java/de/metas/camel/externalsystems/core/AppConfiguration.java

```diff
@@ -61,19 +64,44 @@ public void auditEventNotifier()
 				.addEventNotifier(new AuditEventNotifier(producerTemplate()));
 	}
 
+	@PostConstruct
+	public void authorizationTokenNotifier()
+	{
+		final String defaultAuthToken = context.getEnvironment().getProperty(ExternalSystemCamelConstants.MF_API_AUTHORIZATION_TOKEN_PROPERTY);
+		// note that calling producerTemp8late() here does *not* mean wh create an additional instance. See https://docs.spring.io/spring-framework/docs/current/reference/html/core.html#beans for details
+		camelContext.getManagementStrategy()
+				.addEventNotifier(new MetasfreshAuthorizationTokenNotifier(context, producerTemplate(), defaultAuthToken));
+	}
+
 	@Bean
 	CamelContextConfiguration contextConfiguration()
 	{
 		return new CamelContextConfiguration()
 		{
 			@Override
-			public void beforeApplicationStart(final CamelContext camelContext) {}
+			public void beforeApplicationStart(final CamelContext camelContext)
+			{
+				camelContext.setAutoStartup(false);
+			}
 
 			@Override
 			public void afterApplicationStart(final CamelContext camelContext)
 			{
-				context.getBean(ProducerTemplate.class)
-						.sendBody("direct:" + HANDLE_EXTERNAL_SYSTEM_SERVICES_ROUTE_ID,"trigger rest api handler!");
+				try
+				{
+					camelContext.getRouteController().startRoute(CUSTOM_TO_MF_ROUTE_ID);
+					camelContext.getRouteController().startRoute(CUSTOM_FROM_MF_ROUTE_ID);
+
+					context.getBean(ProducerTemplate.class)
+							.sendBody("direct:" + CUSTOM_TO_MF_ROUTE_ID, "trigger external system authentication for metasfresh!");
+
+					// context.getBean(ProducerTemplate.class)
+					// 		.sendBody("direct:" + HANDLE_EXTERNAL_SYSTEM_SERVICES_ROUTE_ID, "trigger rest api handler!");
```

Comment:
❗ this should be now called after we get the auth token from metasfresh => a nice way to do that would be to change the `HANDLE_EXTERNAL_SYSTEM_SERVICES_ROUTE_ID` definition, from `direct` to a `from("timer://runOnce?repeatCount=1)` that way, when enabled, it will automatically run.. that way it's completely decoupled from the authorization route

-> also, another thing to consider is the start-up order of the routes, as `HANDLE_EXTERNAL_SYSTEM_SERVICES_ROUTE_ID` needs all the other routes to be up and running.. so to achieve that we can take advantage of `org.apache.camel.model.RouteDefinition#startupOrder` -> and then, in our `CustomRouteController.startAllRoutes()` to take that into account

- even more, to have clear evidence of the ordering, I suggest we add an Enum for this rather than point blank setting some integers